src/sentry/static/sentry/app/views/settings/account/accountSecurity/components/recoveryCodes.jsx

```diff
@@ -34,36 +35,70 @@ class RecoveryCodes extends React.Component {
 
     if (!isEnrolled || !codes) return null;
 
+    let formattedCodes = codes.join('\n');
+
+    let printCodes = () => {
+      let iframe = window.frames.printable;
+      iframe.document.write(codes.join('<br>'));
+      iframe.print();
+      iframe.document.close();
+    };
+
     return (
       <Panel css={{marginTop: 30}}>
         <PanelHeader>
```

Comment:
I would add a `hasButtons` prop to this component so the header size isn't as big (and more consistent with other headers that have buttons).